src/main/java/org/dswarm/graph/batch/SimpleNeo4jProcessor.java

```diff
@@ -104,30 +67,13 @@ public void handleSubjectDataModel(final Map<String, Object> subjectNodeProperti
 	}
 
 	@Override
-	public void addStatementToIndex(final long relId, final String statementUUID) {
-
-		statementUUIDs.add(relId, MapUtil.map(GraphStatics.UUID, statementUUID));
-	}
-
-	@Override
 	public Optional<Long> getResourceNodeHits(final String resourceURI) {
 
 		return getNodeIdFromResourcesIndex(resourceURI);
 	}
 
-	@Override
-	public void flushStatementIndices() {
-
-		super.flushStatementIndices();
-
-		statementUUIDs.flush();
-	}
-
-	@Override
-	protected void clearTempStatementIndices() {
-
-		super.clearTempStatementIndices();
+	@Override protected String putSaltToStatementHash(final String hash) {
 
-		tempStatementUUIDsIndex.clear();
+		return hash;
```

Comment:
which one? putSaltToStatementHash? - In SimpleNeo4jProcessor this method is somehow useless. In DataModelNeo4jProcessor the data model uri is added to the hash.